lists/finding_list_4d5a_machine.csv

```diff
@@ -0,0 +1,4 @@
+ID,Category,Name,Method,MethodArgument,RegistryPath,RegistryItem,ClassName,Namespace,Property,DefaultValue,RecommendedValue,Operator,Severity
```

Comment:
Is this a testing list or are you planning to build your own list? - an idea that I would support :)

Reply:
This was initially a testing list, but I plan on building my own. Since my addition of PowerShell Module Logging resulted in some of your finding IDs changing in your finding list, I decided to include my finding list in the pull request so if you decided you wanted to only have PowerShell Script Logging in your finding list, I could have Module Logging in my finding list.